ansible/roles_ocp_workloads/ocp4_workload_integreatly/files/threescale/create-tenant.yml

```diff
@@ -106,7 +106,9 @@
     until: _create_tenantdetails_secret is succeeded
 
   - set_fact:
-      _sso_redirect_uris: "{{ (_action_get_client.resources[0].spec.client.redirectUris | default([])) + [(_tenant_host + '/*' | string)] }}"
+      _sso_redirect_uris: >-
+        {{ (_action_get_client.resources[0].spec.client.redirectUris | default([]))
+        + [(_tenant_host + '/*' | string)] }}
```

Comment:
Let's fix this weird bit to use the proper jinja string concatenation operator, `~`, then it can be on one line:

```
      _sso_redirect_uris: >-
￼        {{ _action_get_client.resources[0].spec.client.redirectUris | default([]) + [(_tenant_host ~ '/*')] }}
```